index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="stylesheet" href="./styles/utilities.css" />
+    <link rel="stylesheet" href="./styles/styles.css" />
+    <title>Mnemba Chambuya</title>
+  </head>
+  <body>
+    <div class="container">
+      <nav>
+        <div class="logo">
+          <a href="#home"><h4>Chambuya</h4></a>
+        </div>
+
+        <i class="fas fa-bars fa-3x"
+          ><input type="checkbox" name="baga-state" id="baga-state"
+        /></i>
+
+        <ul class="nav-links">
+          <li class="nav-link"><a href="">Portfolio</a></li>
+          <li class="nav-link"><a href="">About</a></li>
+          <li class="nav-link"><a href="">Contact</a></li>
+        </ul>
+      </nav>
+      <header class="headline">
+        <hgroup>
+          <h2>Mnemba Chambuya</h2>
+          <h4>Front-end Software Developer</h4>
```

Comment:
- [ ] [REQUIRED] As per the project requirements, you are supposed to stick to the design as much as possible. This includes the font and colors (background and font) defined for the template. please add the font-weight to 500 to match the original design